### PR TITLE
Conference contracts module, conference projection (and tests), move files

### PR DESCRIPTION
**Pull request by AlexanderLitus**
Depends on [#6](https://github.com/SpineEventEngine/Lobby/pull/6).

### CODE REVIEW

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL (depends on [#6](https://github.com/SpineEventEngine/Lobby/pull/6)).

File: common/src/main/proto/lobby/contracts/common/common.proto [hunk 1]
```diff
@@ -21,10 +21,14 @@ syntax = "proto3";
 
 package lobby.contracts.common;
 
-option java_package="lobby.contracts.common";
+option java_package="sample.lobby.contracts.common";
```
Comment (alexander-yevsyukov) [inline common/src/main/proto/lobby/contracts/common/common.proto]: Let's have the following package for all samples: `org.spine3.samples.*`. 
And `lobby` would be one of them.

Comment (alexander-yevsyukov) [inline common/src/main/proto/lobby/contracts/common/common.proto]: We need another name for `contracts` package.

File: registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        final Iterable<SeatType> result = filter(seatTypes, new Predicate<SeatType>() {
+            @Override
+            public boolean apply(@Nullable SeatType input) {
+                return (input != null) && input.getId().equals(id);
+            }
+        });
+        return ImmutableList.copyOf(result);
+    }
+
+    protected static Conference convert(sample.lobby.contracts.conference.Conference c) {
```
Comment (alexander-yevsyukov) [inline registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java]: Why FQN here?

File: registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+import static sample.lobby.registration.Conference.PublishingStatus.PUBLISHED;
+
+/**
+ * The projection of a conference.
+ *
+ * @author Alexander Litus
+ * @see Projection
+ */
+@SuppressWarnings("TypeMayBeWeakened")
+public class ConferenceProjection extends Projection<ConferenceId, Conference> {
```
Comment (alexander-yevsyukov) [inline registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java]: The name isn't clear. What does it project?

Comment (AlexanderLitus) [inline registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java]: Added docs.

File: registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java (hunk 2 above)
Comment (AlexanderLitus) [inline registration/src/main/java/sample/lobby/registration/projection/ConferenceProjection.java]: It is the conference from another module. I'll extract it from both these modules to one.

File: build.gradle [hunk 4]
```diff
@@ -42,10 +46,9 @@ subprojects {
...
 
     dependencies {
-
         // TODO:2015-11-16:alexander.litus: change local dependency to Maven central repository dependency
         // Spine
-        compile group: 'org.spine3', name: 'core-java', version: '0.1', changing: true
+        compile group: 'org.spine3', name: 'core-java', version: '0.2', changing: true
 
         // Guava
         compile group: 'com.google.guava', name: 'guava', version: '19.0-rc2'
```
Comment (alexander-yevsyukov) [inline build.gradle]: Please Guava version to 19.0. It's released already.

File: build.gradle [hunk 5]
```diff
@@ -42,10 +46,9 @@ subprojects {
     }
 
     dependencies {
-
         // TODO:2015-11-16:alexander.litus: change local dependency to Maven central repository dependency
```
Comment (alexander-yevsyukov) [inline build.gradle]: Please document WHEN this change should be made.

File: settings.gradle [hunk 6]
```diff
@@ -12,6 +15,5 @@ project(':payment-contracts').projectDir = "$rootDir/payment-contracts" as File
 include ':registration-contracts'
 project(':registration-contracts').projectDir = "$rootDir/registration-contracts" as File
 
-include ':common'
-project(':common').projectDir = "$rootDir/common" as File
-
+include ':conference-contracts'
```
Comment (alexander-yevsyukov) [inline settings.gradle:18]: Have contracts as parts of corresponding modules coming under `contract` package in both proto and java parts of the code. It's OK to do this in the PR #7, if you'd like.

**Comment by alexander-yevsyukov** [comment]
Please address the recent comments, and then merge.

**Pull request merged**